integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -89,25 +91,27 @@ def get_automatically_saved_pages(self):
         :return: Dictionary containing the context for auto saved pages
         :rtype: dict
         """
-        last_versions = (
-            PageTranslation.objects.filter(
-                language__slug=self.request.region.default_language.slug,
-                page__id__in=Subquery(
-                    self.request.region.non_archived_pages.values("pk")
-                ),
+        if self.request.region.default_language is not None:
+            last_versions = (
+                PageTranslation.objects.filter(
+                    language__slug=self.request.region.default_language.slug,
+                    page__id__in=Subquery(
+                        self.request.region.non_archived_pages.values("pk")
+                    ),
+                )
+                .order_by("page__id", "language__id", "-version")
+                .distinct("page__id", "language__id")
             )
-            .order_by("page__id", "language__id", "-version")
-            .distinct("page__id", "language__id")
-        )
-        automatically_saved_pages = PageTranslation.objects.filter(
-            id__in=Subquery(last_versions.values("pk")),
-            status=status.AUTO_SAVE,
-        ).all()
+            automatically_saved_pages = PageTranslation.objects.filter(
+                id__in=Subquery(last_versions.values("pk")),
+                status=status.AUTO_SAVE,
+            ).all()
```

Comment:
```suggestion
            )
```
Adding `.all()` after `.filter(...)` has no effect. Same goes for `.all()` after '.distinct(...)` above.